spec/models/album_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Album, type: :model do
+  it { should belong_to(:uploader) }
+
+  it { should have_valid(:title).when('Revolver') }
+  it { should have_valid(:title).when('( )') }
+
+  it { should_not have_valid(:title).when(nil, '') }
+
+  it { should have_valid(:date_released).when(Date.today) }
+  it { should have_valid(:date_released).when(nil) }
+end
```

Comment:
:heart: I love seeing these unit tests -- even if they've only got the `shoulda` matchers in them now, they're a great storehouse for later methods/logic that may get added, which should be tested (and can be, right here.)